pkg/stockings/keygen.go

```diff
@@ -0,0 +1,20 @@
+package stockings
+
+import (
+	"math/rand"
+	"os"
+	"strings"
+	"time"
+)
+
+var keys []string
+
+func InitKeygen() {
+	keys = strings.Split(os.Getenv("IEX_TOKEN"), ",")
+}
+
+func getKey() string {
+	rand.Seed(time.Now().UnixNano()) //need this otherwise it only returns one value lmao
```

Comment:
lmao can't we just have a counter and do round robin 🤔

Reply:
random uniformly distributed :)